src/app/components/Transactions/index.tsx

```diff
@@ -73,13 +74,17 @@ export const Transactions: FC<TransactionsProps> = ({
       ? [{ content: (<LockIcon htmlColor={COLORS.grayMedium} />) as unknown as string }]
```

Comment:
`canHaveEncryption` should be hidden in verbose mode too

Reply:
Yes, makes sense.